flamingpy/utils/viz.py

```diff
@@ -284,6 +318,110 @@ def draw_EGraph(
     return fig, ax
 
 
+def draw_EGraph_plotly(
+    egraph,
+    color_nodes=False,
+    color_edges=False,
+    label="coordinates",
+    title=None,
+    legend=False,
+    show_axes=True,
+    **kwargs,
+):
+    """Draw the graph state represented by the EGraph with plotly. #TODO: doc
```

Comment:
Is updating this docstring still a `#TODO`?

Reply:
nope 👍🏼 I'll remove the note